modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-disable.adoc

```diff
@@ -0,0 +1,68 @@
+= rpk cluster partitions disable
+:description: rpk cluster partitions disable
+
+Disable partitions of a topic.
+
+You may disable all partitions of a topic using the `--all` flag or you may select a set of topic/partitions to disable with the `--partitions` or `-p` flag.
+
+The partition flag accepts the format `{namespace}/{topic}/[partitions...]` where namespace and topic are optional parameters. 
```

Comment:
```suggestion
The partition flag accepts the format `<namespace>/<topic>/[partitions...]` where namespace and topic are optional parameters. 
```

Reply:
this is our standard for placeholders. Using `{}` causes Asiidoc to think they are attributes, so we'd need to escape them.